src/hooks/useSelectedTransactionsActions.ts

```diff
@@ -136,6 +154,16 @@ function useSelectedTransactionsActions({
             },
         });
 
+        options.push({
+            text: 'Move Expenses',
+            icon: Expensicons.DocumentMerge,
+            value: 'MOVE',
```

Comment:
Let's use a const for this, probably declared in `CONST.REPORT.SECONDARY_ACTIONS.MOVE_TRANSACTIONS`

Reply:
I'll declare as const on top of the file because it's ignored anyway